modules/stats.php

```diff
@@ -151,14 +151,13 @@ function stats_template_redirect() {
 	$data = stats_array( compact( 'v', 'j', 'blog', 'post', 'tz' ) );
 
 	$stats_footer = <<<END
+<script type='text/javascript' src='$http://stats.wp.com/e-$week.js' async defer></script>
```

Comment:
Is there a reason that we don't just pass the whole url through a `set_url_scheme()` call and let that handle the http/https variants?